dockerfiles/Dockerfile.arm64v8

```diff
@@ -21,72 +9,103 @@ ENV FLB_VERSION 1.8.0
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt update && \
-    apt install -y --no-install-recommends \
+RUN dpkg --add-architecture arm64
+RUN apt update && apt install -y --no-remove --no-install-recommends \
     build-essential \
     g++-aarch64-linux-gnu \
     gcc-aarch64-linux-gnu \
-    curl \
     ca-certificates \
     pkg-config \
     cmake \
     make \
-    tar \
     flex \
-    bison
+    bison \
+    dpkg-dev \
+    libssl-dev:arm64 \
+    libsasl2-dev:arm64 \
+    libsystemd-dev:arm64 \
+    libzstd-dev:arm64 \
+    zlib1g-dev:arm64 \
+    libpq-dev:arm64
 
-COPY --from=dep /usr/lib/aarch64-linux-gnu /usr/lib
-COPY --from=dep /lib/aarch64-linux-gnu /lib
-COPY --from=dep /usr/include /usr/include
+# postgresql-server-dev-11:arm64 has conflicts if installed using apt. hack here to manually force install with dpkg
+WORKDIR /tmp
+RUN apt download postgresql-server-dev-11:arm64
+RUN dpkg --force-all -i postgresql-server-dev-11_11.10-0+deb10u1_arm64.deb
 
 RUN mkdir -p /fluent-bit/bin /fluent-bit/etc /fluent-bit/log /tmp/src/
 COPY . /tmp/src/
 RUN rm -rf /tmp/src/build/*
-
 WORKDIR /tmp/src/build/
+
 RUN cmake -DFLB_RELEASE=On \
-    -DFLB_TRACE=Off \
-    -DFLB_JEMALLOC=On \
-    -DFLB_TLS=On \
-    -DFLB_SHARED_LIB=Off \
-    -DFLB_EXAMPLES=Off \
-    -DFLB_HTTP_SERVER=On \
-    -DFLB_IN_SYSTEMD=On \
-    -DFLB_OUT_KAFKA=On \
-    -DFLB_OUT_PGSQL=On \
-    -DCMAKE_TOOLCHAIN_FILE=/tmp/src/cmake/linux-arm64.cmake ../
+          -DFLB_TRACE=Off \
+          -DFLB_JEMALLOC=On \
+          -DFLB_TLS=On \
+          -DFLB_SHARED_LIB=Off \
+          -DFLB_EXAMPLES=Off \
+          -DFLB_HTTP_SERVER=On \
+          -DFLB_IN_SYSTEMD=On \
+          -DFLB_OUT_KAFKA=On \
+          -DFLB_OUT_PGSQL=On \
+          -DCMAKE_TOOLCHAIN_FILE=/tmp/src/cmake/linux-arm64.cmake ../
 
-RUN make -j $(getconf _NPROCESSORS_ONLN)
+RUN make #-j $(getconf _NPROCESSORS_ONLN)
```

Comment:
why are you changing this?

Reply:
Fixed. It was commented out for testing.